068-kafkabrige-consumer-producer.md

```diff
@@ -0,0 +1,80 @@
+# Enhance KafkaBridge resource with consumer inactivity timeout and HTTP consumer/producer parts enablement
+
+Providing support in the Strimzi Kubernetes Operator for following properties supported in the Strimzi HTTP Kafka Bridge:
+* `http.timeoutSeconds`   - For deleting inactive consumers after a timeout (disabled by default).
+* `http.consumer.enabled` - To enable/disable the HTTP consumer part (enabled by default).
+* `http.producer.enabled` - To enable/disable the HTTP producer part (enabled by default).
+
+## Current situation
+
+Properties are not yet supported in the Strimzi Kubernetes Operator.
+
+## Motivation
+
+Raised in the discussion here: https://github.com/strimzi/strimzi-kafka-operator/issues/8732 and triaged on 29.6.2023: We should enable the configuration of these fields. But how should the API look like? Should have a proposal to clarify the API changes.
+
+## Proposal
+
+This proposal suggests based on discussions here: https://github.com/strimzi/strimzi-kafka-operator/pull/9820 to add the `http enablement` for the `consumer` and `producer` in their respective sessions, i.e. `spec.consumer.enabled` and `spec.producer.enabled`, as well as adding `timeoutSeconds` to the `http` properties.
+
+Suggestion:
+
+```yaml
+apiVersion: "kafka.strimzi.io/v1beta2"
+kind: "KafkaBridge"
+metadata:
+  name: "test-kafka-bridge"
+spec:
+  replicas: 1
+  image: "my-test-image"
+  bootstrapServers: "my-cluster-kafka:9092"
+  http:
+    timeoutSeconds: 60
+  consumer:
+    enabled: true
```

Comment:
I guess this (and the producer one) defaults to true for backward compatibility? It would be worth mentioning it in the text either here or in the compatibility section.

Reply:
Sure, that makes sense :)